elephant/spade.py

```diff
@@ -732,36 +801,39 @@ def _fast_fca(context, min_c=2, min_z=2, max_z=None,
         The size (number of bins) of the sliding window used for the
         analysis. The maximal length of a pattern (delay between first and
         last spike) is then given by winlen*binsize
-        Deafault: 1
+        Default: 1
     min_neu: int (positive)
          Minimum number of neurons in a sequence to considered a
          potential pattern.
          Default: 1
 
-    Returns:
+    Returns
     --------
-    returns:
-    * If report != '#':
+    * If report == 'a':
```

Comment:
I would change the format here. To something like: 
description 
then the if else under the description

Reply:
I realized it was just wrong, I updated according to the function concept_mining, which has the same output